### DISCUSSION TITLE
DSS objects and Signature

### DISCUSSION THREAD

**Discussion by Sidxzx** [pdfcpu/pdfcpu#803] [Q&A] [answered]
Hello,
Is there a way to extract DSS content (CRLs, Certs and OCSPs responses) ? is it also possible to extract the signature from a signed PDF ?

**Comment by hhrutter** [comment, marked as answer]
`DSS` seems to be an optional PDF 2.0 feature around digital signatures.
Supporting PDF 2.0 is an ongoing mission and this will be tackled as part of #168.

**Comment by Sidxzx** [comment]
Alright thank you !
